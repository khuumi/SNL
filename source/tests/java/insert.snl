start first:
    lst is [1,2,3,4]
    do show to lst
    do insert to lst, "inserted", 1
    do show to lst
    do insert to lst, "second", 3
    do show to lst
done
