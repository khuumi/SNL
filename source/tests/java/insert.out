[ 1, 2, 3, 4 ]
[ 1, inserted, 2, 3, 4 ]
[ 1, inserted, 2, second, 3, 4 ]
